### PR TITLE
Added dbeaver.dbeaver version 21.1.2

### PR DESCRIPTION
**Pull request by ImJoakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19818)

### CODE REVIEW

**Comment by ImJoakim** [comment]
```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget install -m "C:\Users\WDAGUtilityAccount\Desktop\manifests\d\dbeaver\dbeaver\21.1.2"
Found DBeaver Community Edition [dbeaver.dbeaver]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
  ██████████████████████████████  90.0 MB / 90.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

**Comment by ImJoakim** [comment]
image

File: manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+PackageVersion: 21.1.2
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
+  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
+    Custom: /currentuser
```
Comment (OfficialEsco) [inline manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.installer.yaml:14]:
```suggestion
- Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://github.com/dbeaver/dbeaver/releases/download/21.1.2/dbeaver-ce-21.1.2-x86_64-setup.exe
  InstallerSha256: 2073B8DEE6EEF6CB1F1BC6B9D6F8F56E4CDCE83DF8ADEB06D3C1E0EFB61C42BA
  InstallerSwitches:
    Custom: /ALLUSERS
```
Only reviewer who will correct this 🙄 Needed for compatibility, at least when the default installation mode is for all users

File: manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+PackageIdentifier: dbeaver.dbeaver
+PackageVersion: 21.1.2
+PackageLocale: en-US
+Publisher: DBeaver Corp
+Author: DBeaver Corp
+PackageName: DBeaver
+PackageUrl: https://dbeaver.io
+License: Apache License v2
+LicenseUrl: https://dbeaver.io/product/dbeaver_license.txt
+ShortDescription: dbeaver.dbeaver
```
Comment (OfficialEsco) [inline manifests/d/dbeaver/dbeaver/21.1.2/dbeaver.dbeaver.locale.en-US.yaml:13]:
```suggestion
ShortDescription: Free multi-platform database tool for developers, database administrators, analysts and all people who need to work with databases.
```
Issue from 0.1.0 to 1.0.0 Manifest Schema Update

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
You can also add the winget install log and ARP screenshot in the top comment so its more noticeable 😁

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**